src/components/ZetkinPerson.tsx

```diff
@@ -14,9 +14,11 @@ const ZetkinPerson: React.FunctionComponent<{
             <Avatar src={ orgId ? `/api/orgs/${orgId}/people/${id}/avatar` : '' } />
             <Box alignItems="start" display="flex" flexDirection="column" justifyContent="center" ml={ 1 }>
                 <Typography variant="body1">{ name }</Typography>
-                { subtitle && (
-                    <Typography variant="body2">{ subtitle }</Typography>
-                ) }
+                { typeof subtitle === 'string' ?
+                    <Typography>{ subtitle }</Typography>
```

Comment:
This needs the `variant="body2"` bit back in it. I will make a commit with this fix :)